lib/Minz/Log.php

```diff
@@ -20,11 +20,14 @@ class Minz_Log {
 	 * @throws Minz_PermissionDeniedException
 	 */
 	public static function record ($information, $level, $file_name = null) {
-		try {
-			$conf = Minz_Configuration::get('system');
-			$env = $conf->environment;
-		} catch (Minz_ConfigurationException $e) {
-			$env = 'production';
+		$env = getenv('FRESHRSS_ENV');
```

Comment:
@marienfressinaud You might want to refactor that at some point, to avoid FreshRSS logic in Minz. And if I remember correctly, your said you would re-think a bit the Minz configuration logic, so I leave that to later :-)

Reply:
Related to:

https://github.com/FreshRSS/FreshRSS/blob/6cedeeeae51f5e5220d7f786fb801951811bb743/lib/Minz/FrontController.php#L118-L122

Reply:
Indeed, and I've started [something in Webubbub](https://github.com/flusio/Webubbub/blob/master/lib/Minz/src/Configuration.php). I plan to import/adapt some of the code I wrote for this project to FRSS (it's a kind of v2 for Minz, it allowed me to test other approaches :))

Reply:
Please try to keep it light and refrain from premature abstractions ;-)